Desktop/main.cpp

```diff
@@ -314,6 +314,8 @@ int main(int argc, char *argv[])
 	// This does slow down QML quite a bit and disables gradients
 	// qputenv("QT_QUICK_BACKEND", "software");
 	//Turned this off because it causes some weird resizing problems QCoreApplication::setAttribute(Qt::AA_UseOpenGLES); //might fix weirdlooking QML on Windows when using not-so-goo drivers? ( https://github.com/jasp-stats/jasp-desktop/issues/2669 )
+	
+	setlocale(LC_ALL, ".65001"); //use utf8
```

Comment:
Why not use ".UTF8" as locale?

Reply:
Because this is windows and that is what it is called right?
Where do you see ".UTF8" as a possible codepage? (as in, if you can show me some docs that state it is acceptable on windows and will gladly change it cause it looks better but I didnt encounter that anywhere)

Reply:
I think docs [here](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/setlocale-wsetlocale?view=msvc-170) is helpful😀

Reply:
This doc just says that ".UTF-8" is the codepage...